src/rgw/const.py

```diff
@@ -141,6 +141,24 @@
 UTF_ENCODING = 'utf-8'
 MOTR_MY_FID = f'{SVC_SECTION}>motr my fid'
 
+# RGW resource limit keys
+SVC_CPU_MIN_VAL_LIMIT = '250m'
+SVC_MEM_MIN_VAL_LIMIT = '128Mi'
+SVC_LIMIT_KEY = f'cortx>{COMPONENT_NAME}>limits'
+SVC_LIMIT_NUM_SERVICES= f'{SVC_LIMIT_KEY}>num_services'
+SVC_LIMIT_NAME = f'{SVC_LIMIT_KEY}>services[%s]>name'
+SVC_LIMIT_CPU_MIN_KEY = f'{SVC_LIMIT_KEY}>services[%s]>cpu>min'
+SVC_LIMIT_MEM_MIN_KEY = f'{SVC_LIMIT_KEY}>services[%s]>memory>min'
+SVC_RESOURCE_LIMIT_MEM_VAL_SUFFIXES = ['K', 'Ki', 'Kib', 'M', 'Mi', 'Mib', 'G', 'Gi', 'Gib']
+SVC_RESOURCE_LIMIT_MEM_VAL_SIZE_MAP = {
+        "K": 1024, "M": 1024*1024, "G": 1024*1024*1024,
```

Comment:
For Ki, Kib units (& alike) why cant we match values against K, we dont need keep duplicated values for possible variants of same unit